android/java/apk_for_test.flags

```diff
@@ -86,6 +86,7 @@
 -keep class org.chromium.chrome.browser.search_engines.settings.SearchEngineAdapter {
     *** getSearchEngineSourceType(...);
     *** sortAndFilterUnnecessaryTemplateUrl(...);
+    *** didSearchEnginesChange(...);
```

Comment:
And we probably also need `computeStartIndexForRecentSearchEngines` here to pass tests in release

Reply:
fixed at 
https://github.com/brave/brave-core/pull/14402/commits/8a59b00bf5ecaa19b612d9b1e74b86e815cbe5ad#diff-328021dc35ff453fa79b8f57c279b5365d6c1a095a10ebd8bb8f98a67cdd8865R90